simplecpp.cpp

```diff
@@ -1746,20 +1746,15 @@ static bool realFileName(const std::string &f, std::string *result)
     if (!alpha)
         return false;
 
-    // Convert char path to CHAR path
-    std::vector<CHAR> buf(f.size()+1U, 0);
```

Comment:
must look a little at this. I thought CHAR could be 16-bit also.

I want to handle non-plain-ASCII filenames also such as åäö.cpp etc. But I don't really remember how well tested and well implemented this is.

Reply:
https://msdn.microsoft.com/de-de/en-en/library/windows/desktop/dd374131(v=vs.85).aspx indicates that CHAR is 8-bit always.